File: factorial.js
function factorial(n) {
    if (n<0) {
        return("Negatives don't have factorial")
    }
    else if (n==0 || n==1) {
        return (1)
    }
    else {
        while(n>1) {
            return (n*factorial(n-1))
        }
    }
}
number = prompt("Enter number")
console.log(factorial(number))

File: factorial.py
def factorial(n):
    if (n<0): 
        return ("ERROR - Negatives can't have factorials !!")
    if (n==0 or n==1):
        return 1
    while (n>1):
        return n*factorial(n-1)

number = int(input("Enter number"))
print (factorial(number))
